templates/plugin/content/package.json

```diff
@@ -3,6 +3,10 @@
   "description": "A Fusion.js plugin",
   "version": "0.0.0",
   "license": "MIT",
+  "files": [
```

Comment:
We re-write this in the scaffolder, so this ends up being a no-op.  Necessary for `yarn install` for the template to succeed here though.